packages/manager/modules/vps/src/vps/translations/Messages_fr_FR.json

```
@@ -99,6 +99,16 @@
   "vps_dashboard_monitoring_sla": "Monitoring SLA",
   "vps_dashboard_monitoring_sla_ips": "IPs monitoring SLA",
   "vps_dashboard_monitoring_sla_ips_tooltip": "Pour le bon fonctionnement de nos outils de monitoring, merci d'autoriser ces plages d'IPs dans votre firewall.",
+  "vps_dashboard_snapshot_download_modal_title": "Download Snapshot",
+  "vps_dashboard_snapshot_download_modal_summary": "Informations",
+  "vps_dashboard_snapshot_download_modal_service_name": "Nom du service",
+  "vps_dashboard_snapshot_download_modal_snapshot_size": "Taille",
+  "vps_dashboard_snapshot_download_modal_snapshot_creation_date": "Date de création du snapshot",
+  "vps_dashboard_snapshot_download_modal_snapshot_link_expiration_date": "Date d'expiration du lien de téléchargement",
+  "vps_dashboard_snapshot_download_modal_snapshot_loading_message": "Génération de la commande de téléchargement du snapshot ...",
+  "vps_dashboard_snapshot_download_modal_snapshot_curl_success": "Commande de téléchargement générée avec succes.",
+  "vps_dashboard_snapshot_download_modal_snapshot_curl_info": "Veilliez copier la command CURL ci-dessus et l'exécuter sur votre terminal. Pour en savoir plus sur CURL <a href=\"{{curlDocumentationLink}}\" target=\"_blank\">Cliquez ici</a>.",
```

```suggestion
  "vps_dashboard_snapshot_download_modal_snapshot_curl_info": "Veuillez copier la commande CURL ci-dessus et l'exécuter sur votre terminal. Pour en savoir plus sur CURL <a href=\"{{curlDocumentationLink}}\" target=\"_blank\">Cliquez ici</a>.",
```
and you can remove / move the html code

---

Done.